src/codecs/jats/index.ts

```diff
@@ -3,96 +3,89 @@
  */
```

Comment:
Not sure if you rebased before refactoring this file, seems to have lost some of the changes from the `jats-refactor` branch.

Reply:
Yes, I think that has something to do with the need to update a lot of the JATS test snapshats.

Reply:
hmmm, thought I was careful not to mangle anything, but clearly I was mistaken. Will give it another go